debezium-server/debezium-server-pubsub/src/test/java/io/debezium/server/pubsub/PubSubTestConfigSource.java

```diff
@@ -14,10 +14,13 @@
 
 public class PubSubTestConfigSource extends TestConfigSource {
 
+	public static final String PUB_SUB_ADDRESS = System.getenv("PUBSUB_EMULATOR_HOST");
+	
     public PubSubTestConfigSource() {
         Map<String, String> pubsubTest = new HashMap<>();
 
         pubsubTest.put("debezium.sink.type", "pubsub");
+        pubsubTest.put("debezium.sink.pubsub.address", PUB_SUB_ADDRESS);
```

Comment:
This should be set only if the env var is present as in real deployment the missing setting will not turn into `null` value.